pom.xml

```diff
@@ -138,6 +138,22 @@
 		</dependency>
 
 		<dependency>
```

Comment:
I missed this one last time I checked, but flink-ml is already imported at line 108.